### DISCUSSION TITLE
Running on vercel --> custom domain --> have access to chats on all my devices?

### DISCUSSION THREAD

**Discussion by bhuisman1992** [lobehub/lobehub#1487] [General Question]
Hi,

I've just installed this on vercel and i've attachted it to a custom domain. Maybe this is a pretty dumb question but, can i have the same chats on my computer as well as on my mobile phone.. or how does this work?

My initial thoughts were: you set this up and the chats will be saved somewhere so i can access it on whichever device i want. Would be great i someone can clarify this for me.

**Comment by cy948** [comment]
see RFC 009 https://github.com/lobehub/lobe-chat/discussions/368
